roles/system/templates/services/system-sphinx.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ sphinx.env.thinking.service | default(sphinx_env_thinking_service) }}
+  namespace: "{{ meta.namespace }}"
+  labels:
+    name: system-app
```

I guess this is a typo:
```suggestion
    app: 3scale-api-management
```